test/built-ins/Promise/prototype/finally/this-value-non-promise.js

```diff
@@ -11,7 +11,9 @@ features: [Promise.prototype.finally]
 var called = false;
 var p = new Proxy(Promise.resolve(), {});
 var oldThen = Promise.prototype.then;
-Promise.prototype.then = () => { called = true; };
+Promise.prototype.then = () => {
```

Comment:
Oops, this probably shouldn’t have been an arrow function.

Reply:
surely not, but also not really a thing to be fixed on this PR. After this work I should take a look for all the occurrences of arrow fns in the built-ins folder.